mscore/inspector/inspector_tuplet.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>304</width>
-    <height>288</height>
+    <width>421</width>
```

Comment:
Usually multiplies of 12 are used.

Did you copy it from somewhere or it's a 'by eye' value?

Reply:
@vpereverzev I haven't noticed that "tradition". But yes, Qt Designer did it. I suppose this doesn't really matter because inspector widgets always display fixed width.

Reply:
I usually just avoid making such changes part of my commits, restringi it to thosie I know I want changed

Reply:
here height changed by 15, sonsistent with the 5*3 further down. Doesn't explain the width change though

Reply:
except that it decreased by 15, not increased?

Reply:
Understood. If necessary, I can reduce the width to minimum and the height to a reasonable smaller value (although I don't see why it isn't reasonable now :-) ).

Reply:
IMHO it isn't here

Reply:
Come on, the height doesn't even matter. Only the title button, which will have a fixed size after compliation, is stretching/folding when you change the height.

Reply:
That's what I said. It isn't reasonable to change this, there is no neccessity.

Reply:
@Howard-C, could you please exclude changes to these two lines from the commit? They are not relevant to this commit so it is better to leave them unchanged.

Reply:
They are not relevant to this commit but are still good changes. The old sizes are too small to show the full UI in Qt Creator/Designer, it is only completely shown in MuseScore because it's part of the inspector. The current sizes are the minimum sizes to show the full UI, so they're better in this way.